tools/sandbox/application.py

```diff
@@ -188,39 +208,36 @@ def check() -> str:
     return pyre.check(input)
 
 
-@application.route("/analyze", methods=["POST"])
-def analyze() -> str:
-    input = (
-        request.args.get("input")
-        or request.form.get("input")
-        or request.json.get("input")
-    )
-    use_builtin_pysa_models = bool(
-        request.args.get("use_builtin_pysa_models")
-        or request.form.get("use_builtin_pysa_models")
-        or request.json.get("use_builtin_pysa_models")
-    )
-    model = (
-        request.args.get("model")
-        or request.form.get("model")
-        or request.json.get("model")
-    )
+@socketio.on("analyze", namespace="/analyze")
+def analyze(json) -> None:
+    print(json.get("input", None))
```

Comment:
Presumably we don't want to be printing to stdout here? Is this leftover from debugging?

Reply:
Yeah, I do a bit of hard testing and this is the remainders from such a debugging process. So sorry.